src/seismometer/seismogram.py

```diff
@@ -179,27 +190,27 @@ def target_event(self, event: str):
 
     @property
     def target_cols(self) -> list:
-        return self.config.targets
+        return list(self.config.targets)
 
     @property
     def intervention(self):
```

Comment:
can we add the return type for this property, it is a string, or a new object?

Reply:
Added here and for outcome, these are the display_names as well.